File: sprint1/By languages/C#/lesson5code2.cs
using System;

public class Program
{
    public static void Main()
    {
        var timeStart = DateTime.Now;
        int i = 0;
        while (i < 1000000000)
        {
            // Do nothing
            i++;
        }

        var timeFinish = DateTime.Now;
        TimeSpan timeSpan = timeFinish - timeStart;
        Console.WriteLine(timeSpan.TotalSeconds + " seconds");
    }
}

File: sprint1/By languages/Java/lesson5code2.java
long time_start = System.currentTimeMillis();
long i = 0;
while (i < 1000000000) {
    i++;
}
long time_finish = System.currentTimeMillis();
long time_span = time_finish - time_start;
System.out.println(time_span / 1000.0 + " seconds");
